pages/probes/[id].vue

```diff
@@ -0,0 +1,367 @@
+		:draggable="false"
+		header="Probe details"
+		content-class="!p-0"
+		@after-hide="navigateTo('/probes')"
+	>
+		<div class="relative border-t">
+			<div id="gp-map" class="h-44"/>
+			<div class="absolute inset-x-0 top-2 ml-4 mr-20 flex justify-between dark:text-bluegray-900">
+				<div>
+					Status:<span class="ml-2 font-bold">{{ capitalize(probe.status.replaceAll('-', ' ')) }}</span>
+					<StatusIcon class="ml-2 text-3xs" :status="probe.status"/>
+				</div>
+				<div class="flex items-center">
+					Credits per month:
+					<nuxt-icon class="ml-2 text-green-500" name="coin"/>
+					<span class="ml-2 font-bold text-green-500">+{{ props.credits }}</span>
+				</div>
+				<div>Type:<span class="ml-2 font-bold">{{ probe.hardwareDevice || 'Container' }}</span></div>
+				<div>Version:<span class="ml-2 font-bold">{{ probe.version }}</span></div>
+			</div>
+		</div>
+		<div class="px-5 py-7 dark:text-surface-0">
+			<label for="probeName" class="text-xs">Probe name</label>
+			<InputText
+				id="probeName"
+				v-model="probe.name"
+				class="mt-1 w-full"
+			/>
+			<label for="primary-ip" class="mt-3 inline-block text-xs">Primary IP</label>
+			<div class="relative mt-1">
+				<i class="pi pi-lock absolute right-3 top-2.5 text-bluegray-500"/>
+				<InputText
+					id="primary-ip"
+					v-model="probe.ip"
+					disabled
+					class="w-full bg-transparent dark:bg-transparent"
+				/>
+			</div>
+			<label for="alternative-ips" class="mt-3 inline-block text-xs">Alternative IPs</label>
+			<div class="relative mt-1">
+				<i class="pi pi-lock absolute right-3 top-[13px] text-bluegray-500"/>
+				<AutoComplete
+					id="alternative-ips"
+					v-model="probe.altIps"
+					class="bg-transparent dark:bg-transparent"
+					chip-icon="hidden"
+					multiple
+					disabled
+					:typeahead="false"
+				/>
+			</div>
+			<label for="city" class="mt-3 inline-block text-xs">Location</label>
+			<InputGroup class="mt-1">
+				<InputGroupAddon class="!bg-transparent">
+					<CountryFlag :country="probe.country" size="small"/>
+				</InputGroupAddon>
+				<InputText
+					id="city"
+					v-model="probe.city"
+					class="w-full"
+				/>
+			</InputGroup>
+			<p class="mt-2 text-xs text-bluegray-400">
+				City where the probe is located. If you know that city is wrong it can be changed here: type in the valid city and click save.
+			</p>
+			<label for="tags" class="mt-3 inline-block text-xs">Tags</label>
+			<div v-if="isEditingTags" class="mt-1">
+				<div>
+					<div v-for="(tag, index) in tagsToEdit" :key="index" class="mb-2 flex items-center" :class="{ 'mb-5': !isTagValid(tag.value) }">
+						<Select v-model="tag.uPrefix" class="flex-1" :options="uPrefixes" :scroll-height="'200px'"/>
+						<span class="mx-2">-</span>
+						<span class="relative flex-1">
+							<InputText v-model="tag.value" :invalid="!isTagValid(tag.value)" class="w-full"/>
+							<p v-if="!isTagValid(tag.value)" class="absolute pl-1 text-red-500">Invalid tag</p>
```

Comment:
`<p>` can't be inside of `<span>`; change to `span` as well, or change the parent to a `div`